app/controllers/rails_admin/main_controller.rb

```diff
@@ -120,7 +120,7 @@ def check_for_cancel
     end
 
     def get_collection(model_config, scope, pagination)
-      associations = model_config.list.fields.select { |f| f.type == :belongs_to_association && !f.polymorphic? }.collect { |f| f.association.name }
+      associations = model_config.list.fields.select { |f| f.type.in?([:belongs_to_association, :has_one_association, :has_many_association]) && !f.polymorphic? }.collect { |f| f.association.name }
```

Comment:
```ruby
def get_collection(model_config, scope, pagination)
  associations = _associations_for_collection(model_config)
  # etc
end
def _associations_for_collection(model_config)
  model_config.list.fields.select { |f| f.type == :belongs_to_association && !f.polymorphic? }.collect { |f| f.association.name }
end
```

so that we can customize it?